day7/sunny/index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>HW7</title>
+    <link rel="stylesheet" href="index.css" />
+    <link rel="stylesheet" href="./normalize.css" />
+  </head>
+  <body>
+    <div class="wrapper mobile">
+      <img src="iphone.png" alt="mobile image" class="background" />
+      <div class="home-splash-container">
+        <div class="home-header-container">
+          <div class="home-logo-container">
+            <img src="logo.svg" alt="logo" class="logo" />
+          </div>
+          <div class="home-text-container">
+            <p>Saturn is best experienced</p>
+            <p>on our mobile app</p>
+          </div>
+        </div>
+        <div class="home-footer-container">
+          <button>Download Staurn</button>
+        </div>
+      </div>
+    </div>
+
+    <nav class="home-wrapper">
+      <div class="home-logo">
+        <img src="logo.svg" alt="logo" />
+        <p>saturn</p>
+      </div>
+      <div class="home-nav-element-container">
+        <div class="home-circel">
+          <div class="home-time-container">
+            <h6>Period 2</h6>
+            <h3>45 : 27</h3>
+            <p>25% done</p>
+          </div>
+        </div>
+        <div class="home-nav-info-container">
+          <div class="nav-title">
+            <h4>AP Computer Science</h4>
+            <p>Mr. Nessi &sdot; Room 25A &sdot; 9:36-10:43</p>
+          </div>
+          <div class="nav-percentage-bar">
+            <p><span>25%</span> of your day is completed! &#128526;</p>
+            <div class="bar-container">
+              <div class="bar"></div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </nav>
+
+    <section>
+      <div class="home-body">
+        <div class="home-body-header">
+          <div>
+            <p>Belmont High School</p>
+            <h3>Monday, Feb. 24 - <span>A Day</span></h3>
+          </div>
+          <button><img src="arrow.png" alt="arrow" />Log Out</button>
+        </div>
+        <div class="home-body-sections">
+          <div class="section-number taken-class">1</div>
+          <div class="section-class taken-class">
+            <h3>Philosophy</h3>
+            <p>8:00-9:31 &sdot; Mr. Nessi &sdot; Room 25A</p>
+          </div>
+        </div>
+        <div class="home-body-sections">
```

Comment:
do not use plural form if it's a single section

Reply:
adjusted